inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/task/TaskPositionManager.java

```diff
@@ -91,13 +97,15 @@ private Runnable taskPositionFlushThread() {
                         JobProfile jobProfile = jobConfDb.getJobById(jobId);
                         if (jobProfile == null) {
                             LOGGER.warn("jobProfile {} cannot be found in db, "
-                                + "might be deleted by standalone mode, now delete job position in memory", jobId);
+                                    + "might be deleted by standalone mode, now delete job position in memory", jobId);
                             deleteJobPosition(jobId);
                             continue;
                         }
                         flushJobProfile(jobId, jobProfile);
                     }
-                    TimeUnit.SECONDS.sleep(DEFAULT_FLUSH_TIMEOUT);
+                    int flushTime = conf.getInt(AGENT_HEARTBEAT_INTERVAL,
+                            DEFAULT_AGENT_FETCHER_INTERVAL);
```

Comment:
Here the fetcher timeout may be inconsistant with the code

Reply:
In old cold, HeartBeatManager use the same constant(DEFAULT_FLUSH_TIMEOUT) with TaskPositionManager, so i use HEART_BEAT_INTERVAL instead

Reply:
+1